### PR TITLE
Fix make vet fmt

### LINKED ISSUE #6: make vet fmt is broken
```
$ make vet fmt
cd /Users/dims/go/src/git.openstack.org/openstack/openstack-cloud-controller-manager && go vet ./...
# git.openstack.org/openstack/openstack-cloud-controller-manager/pkg/volume/cinder/provisioner
pkg/volume/cinder/provisioner/iscsi_test.go:61:4: err declared but not used
pkg/volume/cinder/provisioner/iscsi_test.go:105:4: err declared but not used
pkg/volume/cinder/provisioner/iscsi_test.go:153:4: err declared but not used
pkg/volume/cinder/provisioner/rbd_test.go:78:4: err declared but not used
vet: typecheck failures
make: *** [vet] Error 2
```

### PR DESCRIPTION
**Pull request by hogepodge**
**What this PR does / why we need it**:

Several tests had unused err variables. Removed them to fix `make vet fmt`

**Which issue this PR fixes**
fixes #6 

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by flaper87** [comment]
lgtm

**Comment by dims** [comment]
one more thing to fix @hogepodge

File: pkg/volume/cinder/provisioner/iscsi_test.go [hunk 1]
```diff
@@ -74,7 +73,7 @@ var _ = Describe("Iscsi Mapper", func() {
 
 		JustBeforeEach(func() {
 			secretName = getChapSecretName(conn, options)
-			source, err = mapper.BuildPVSource(conn, options)
```
Comment (anguslees) [inline pkg/volume/cinder/provisioner/iscsi_test.go]: The idea of the lint is that ignoring the error is a bug - not that you should just ignore the error in a different way :-P

Gingko makes this easy with an assertion like:
`Expect(err).NotTo(HaveOccurred())`

**Comment by hogepodge** [comment]
@dims done

File: pkg/volume/cinder/provisioner/iscsi_test.go (hunk 1 above)
Comment (hogepodge) [reply pkg/volume/cinder/provisioner/iscsi_test.go]: Gotcha. Had a bad feeling about just dropping all those errors, just wasn't sure how to go about it. Thanks.

**Comment by hogepodge** [comment]
Ok, so golint is not available in the test environment. I've updated the Makefile to install it if it isn't available.

**Pull request merged**